api/swagger-spec/v1.json

```diff
@@ -15402,7 +15402,7 @@
       "type": "string",
       "method": "GET",
       "summary": "connect GET requests to proxy of Service",
-      "nickname": "connectGetNamespacedServiceProxy",
+      "nickname": "connectGetNamespacedServiceProxyWithPath",
```

Comment:
it is outside of the scope of this change, I will let it pass for this PR but will think about it.